config/settings/base.py

```diff
@@ -80,6 +80,7 @@
 
 LOCAL_APPS = [
     "medium.users",
+    "medium.profiles",
     # Your stuff: custom apps go here
```

Comment:
Solve the conflict in line 83

Reply:
Resolve conflicts button is disabled i think you should approve the pull request first then i solve conflict.

Reply:
No, it's a protected branch. I think @OsamaRagab520 can disable this from settings.
You can solve the conflict locally or wait until Osama changes the settings.

<img width="960" alt="image" src="https://user-images.githubusercontent.com/49886457/165399315-70beeb07-3958-4a04-9240-c50c94c1aa1f.png">

Reply:
merge master on your branch and solve it locally then push the merging commit 
the pull request will be updated automatically